src/chainparams.cpp

```diff
@@ -435,7 +435,8 @@ class CRegTestParams : public CChainParams {
         assert(genesis.hashMerkleRoot == uint256S("0x4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b"));
 
         vFixedSeeds.clear(); //!< Regtest mode doesn't have any fixed seeds.
-        vSeeds.clear();      //!< Regtest mode doesn't have any DNS seeds.
+        vSeeds.clear();
+        vSeeds.emplace_back("dummySeed.invalid");
```

Comment:
Could use a FQDN such as say `dummy-seed.invalid.` (note the ending dot) be used here to make it unambiguous and thus make sure we don't risk causing a DNS lookup for `dummySeed.invalid.<local domain name as specified in /etc/resolv.conf>`)?

Reply:
good call, updated to `dummySeed.invalid.`